src/views/HashtagPostView/index.tsx

```diff
@@ -81,11 +84,21 @@ const HashtagPostView = ({ pageData, seo }: HashtagProps): JSX.Element => {
       <Container py={4}>
         <Box textAlign="center">
           <Heading>{pageData?.hashtag?.title}</Heading>
-          <Text my={4} maxW="container.md" mx="auto">
-            {pageData?.hashtag?.content}
-          </Text>
+          <Collapse startingHeight={50} in={show}>
+            <Text my={4} maxW="container.md" mx="auto">
+              {pageData?.hashtag?.content}{' '}
+            </Text>
+          </Collapse>
+          <Button
+            bg="transparent"
+            borderColor="transparent"
+            color="primary.500"
+            size="sm"
+            onClick={handleToggle}
+          >
+            {show ? <>{t`less`}</> : <>{t`more`}</>}
+          </Button>
```

Comment:
```suggestion
          <IconButton
            variant="ghost"
            size="sm"
            icon={FaChevronDown}
            onClick={handleToggle}
          />
```

It's ok! I have just a suggestion. In this case we don't worry about button text translations.